zeppelin-interpreter/src/test/resources/conf/interpreter.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
This is for testing. In interpreter.json, I define one interpreter `EchoInterpreter ` for group `test`. And in interpreter-setting.json, I define 2 interpreters `EchoInterpreter` & `DoubleInterpreter`. This is trying to simulate the scenario that we add a new interpreter in a new release. The new interpreter should be able to be detected.